### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by valentynavik**
https://valentynavik.github.io/layout_product-cards/
https://valentynavik.github.io/layout_product-cards/report/html_report/

 `Checklist` to the `Pull Request`:
- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,11 +3,54 @@
...
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
```
Comment (volodymyr-soltys97) [inline src/index.html:14]:
```suggestion
    <link
      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
      rel="stylesheet"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -3,11 +3,54 @@
...
+        src="../src/images/imac.jpeg"
+        class="card-img"
+        alt="альтернативний текст"
+      >
+
+      <h2 class="card-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card-code">Product code: 195434</div>
```
Comment (volodymyr-soltys97) [inline src/index.html:29]: Use a semantic tag everywhere for text

File: src/index.html [hunk 3]
```diff
@@ -3,11 +3,54 @@
...
+        <div class="card-price__text">Price:</div>
+        <div class="card-price__price">$2,199</div>
+      </div>
+
+      <a
+        class="card-button"
+        href="#"
+        data-qa="hover">
+        Buy
+      </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:53]:
```suggestion
      <a
        class="card-button"
        href="#"
        data-qa="hover"
      >
        Buy
      </a>
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:2]: Move box-sizing to the * selector

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+  &-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    border-radius: 5px;
+    box-sizing: border-box;
+    border: 1px solid #00acdc;
+    background-color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:70]: Create a variable for this color and use it everywhere

File: src/styles/blocks/stars.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+
+
+.stars--1 .stars__star:nth-child(-n + 1),
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/stars.scss:20]:
```suggestion
}

.stars--1 .stars__star:nth-child(-n + 1),
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix demo link

**Pull request closed without merging**